airflow-core/src/airflow/ui/src/pages/TaskInstance/ExtraLinks.tsx

```diff
@@ -38,7 +38,7 @@ export const ExtraLinks = () => {
         {Object.entries(data.extra_links).map(([key, value], _) =>
           value === null ? undefined : (
             <Button asChild colorPalette="blue" key={key} variant="surface">
-              <a href={value} rel="noreferrer" target="_blank">
+              <a href={value} rel="noopener noreferrer" target="_blank">
                 {key}
               </a>
             </Button>
```

Comment:
Ahh I see where you got the wrong impression from. Let me fix this in another PR.